<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size :24px;
}

QGroupBox {
background:#333;
border-radius:15px;
}
QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}
QToolButton {
background-color :#CD5C5C;border:none;
 border-radius: 40px;
  min-height: 20px;
  min-width: 20px;
}
QPushButton:hover
{ color:#333;
 border-radius: 15px;
background-color :#49ebff;border:none;
}
 QLabel {
color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Signin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>431</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>281</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>   LOGIN HERE</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
background:transparent;
border:none;
color: white;

}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/imageedit_9_2905762923.png</normaloff>../../Downloads/imageedit_9_2905762923.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
